GFW/convert.py

```diff
@@ -0,0 +1,103 @@
+import os
+import time
+import xarray as xr
+import rioxarray as rxr
+from rasterio.enums import Resampling
```

Comment:
I am not seeing these rasterio imports being used. In your VScode, I would recommend installing the 'Ruff' extension. I have these settings also:

```json
    "ruff.lint.args": [
        "--line-length=88",
        "--extend-select=D403"
    ],
    "ruff.organizeImports": true,
```

This is actually how I learn best practices in python. Ruff will complain is VS code when I do bad things, like leave imports that I don't use.

Reply:
Yeah vscode like makes the import pale and I'll go back and check it manually at the end to remove anything I didn't use. I'll take a looksies at the Ruff extension!!